lib/idy/extension.rb

```diff
@@ -46,7 +46,7 @@ def idy_decode(hash, salt: self.salt)
       def idy_default_salt
         alphabet = Array('a'..'z')
 
-        indexes = name.downcase.split('').map do |char|
+        indexes = name.split("::").last.downcase.split('').map do |char|
```

Comment:
What do you think about let just the `shift` restrict the size, not two side effects, and just ignoring the colon like:

```
safe_letters = name.downcase.split('') - [':']
indexes      = safe_letters.map { |char| alphabet.index(char) + 1 }
```

Reply:
I think if use the `#tr` method is more efficient. LIke this:
```suggestion
        indexes = name..tr(':', '').last.downcase.split.map do |char|
```

The idea here is to remove the `":"` character.

Reply:
@rafaeldev I agree, but in this case, we don't have the `last`, since we don't have the `split` anymore.